lib/guardian/plug.ex

```diff
@@ -117,6 +117,7 @@ defmodule Guardian.Plug do
     case Guardian.encode_and_sign(object, type, new_claims) do
       {:ok, jwt, full_claims} ->
         conn
+        |> Plug.Conn.configure_session(renew: true)
```

Comment:
This doesn't appear to matter and I think it reads a bit clearer this way so I'm going to leave it unless @scrogson or @hassox feels strongly about it.